MantidQt/MantidWidgets/src/MuonFitPropertyBrowser.cpp

```diff
@@ -168,7 +168,8 @@ void MuonFitPropertyBrowser::init() {
   tmp = "bwd";
   addGroupCheckbox(tmp);
   m_periodsToFit = m_enumManager->addProperty("Periods to fit");
-  m_periodsToFitOptions << "1"
+  m_periodsToFitOptions << "All Periods"
```

Comment:
If you are using `All Periods` and the other values only in `MuonFitPropertyBrowser` then it is worth defining them as variables in a `static const`. This way we write the string only once.